app/views/case_logs/_log_filters.erb

```diff
@@ -0,0 +1,32 @@
+<div class="app-filter-layout__filter" tabindex="-1" id="filter-panel">
+    <div class="app-filter" >
+        <div class="app-filter__header">
+            <h2 class="govuk-heading-m">Filters</h2>
+            <button 
+            class="app-filter__close" 
+            type="button" 
+            data-controller="filter"
+            data-action="click->filter#toggleFilter">
+            Close
+            </button>
+        </div>
+        <div class="app-filter__content">
+            <div class="govuk-form-group app-filter__group">
+                <%= form_with url: "/logs/filter", html: { method: :post }, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+                    <%= f.govuk_check_boxes_fieldset :status, legend: { text: "Status", size: "s"} do %>
+                        <div class="govuk-checkboxes govuk-checkboxes--small" data-module="govuk-checkboxes">
+                            <% statuses = {in_progress: "In progress", submitted: "Completed", not_started: "Not started"} %>
```

Comment:
Could we use a helper to derive these from `CaseLog::STATUS`? Such that we only have one source of truth for what valid statuses are?